generateRandomQueries/generate_procedure.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python2
+
+import subprocess
+import random
+
+
+TABLE_PREFIX = "table"
+SYNC_METADA_SQL = "select start_metadata_sync_to_node(nodename, nodeport) from pg_dist_node;"
+CREATE_TABLE = "CREATE TABLE {}(a int, b int);"
+DISTRIBUTE_TABLE = "SELECT create_distributed_table('{}', 'a');"
+INSERT_INTO_TABLE = "INSERT INTO {} SELECT *,* FROM generate_series(1, {});"
+
+SETTINGS = [
+    "citus.replication_model TO 'streaming'",
+    "citus.shard_count TO 128"
+]
+SETTING_STRING = "set {};"
+
+MAX_QUERY_PER_PROCEDURE = 300
+MAX_CALL_AMOUNT = 100000
+MIN_TABLE_SIZE = 1000
```

Comment:
Minor: table size sounds a little confusing since it might mean disk size etc. So, maybe `MIN_TABLE_ROWS` / `MAX_TABLE_ROWS`?

Also, it might be super useful to have very short descriptions of the variables as comments.  For example, it is not very obvious what `REPEAT_AMOUNT ` mean